README.md

```diff
@@ -68,5 +69,5 @@ To run a subset of the tests you can use the `-t` option of Jest to only run tes
 
 `yarn run unit -t group`
 
-Copyright (c) 2019 IronCore Labs, Inc.
+Copyright (c) 2018-present IronCore Labs, Inc.
```

Comment:
```suggestion
Copyright (c) 2020 IronCore Labs, Inc.
```

I think technically we don't need any more than `Copyright (c) IronCore Labs, Inc.` because the years are automatic based on publishing individual files, but `present` doesn't hold up in court. `2019 - 2020` might but isn't necessary.

Reply:
Huh, TIL, thanks.